OrbitClientData/ModuleData.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientData/ModuleData.h"
+
+#include "OrbitBase/Logging.h"
+#include "OrbitClientData/FunctionUtils.h"
+#include "absl/synchronization/mutex.h"
+#include "capture_data.pb.h"
+
+using orbit_client_protos::FunctionInfo;
+
+bool ModuleData::is_loaded() const {
+  absl::MutexLock lock(&mutex_);
+  return is_loaded_;
+}
+
+const FunctionInfo* ModuleData::FindFunctionByRelativeAddress(uint64_t relative_address,
+                                                              bool is_exact) const {
+  absl::MutexLock lock(&mutex_);
+  if (functions_.empty()) return nullptr;
+
+  uint64_t address = relative_address + load_bias();
```

Comment:
ok, I think this introduces a new kind of address we did not have before. So now we have absolute address, elf-address, and relative address. Does it give us any advantages? say if I have an address from the elf file symbol table I now have to calculate load-bias and substract it from the address before passing it to this function?

Reply:
As discussed offline, I added another function called FindFunctionByElfAddress and used it in FindFunctionByRelativeAddress. This should clarify the code.